src/App.tsx

```diff
@@ -43,7 +43,7 @@ class App extends React.Component<{}, State, {}> {
             <Switch>
               <Route exact path="/" render={() => <Redirect to="/home" />} />
               <Route path="/home" render={() => <Home />} />
-              <Route path="/contact-us" render={() => <ContactUs />} />
+              {/* <Route path="/contact-us" render={() => <ContactUs />} /> */}
```

Comment:
I think we can still keep the contact us page as it contains other info that is useful